eslint.config.js

```diff
@@ -4,6 +4,8 @@ import reactHooks from 'eslint-plugin-react-hooks'
 import storybook from 'eslint-plugin-storybook'
 import globals from 'globals'
 import typescript from 'typescript-eslint'
+import prettierPlugin from 'eslint-plugin-prettier'
+import prettierConfig from 'eslint-config-prettier'
```

Comment:
I like prettier, I'm open to considering it. I'm not at all happy with the current eslint situation. But the change belongs in its own PR.

Reply:
Yeah sounds good, I'll revert, and create an issue to add prettier in.